client/src/components/GamePage.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+const Game = ({ me, gameID }) => {
```

Comment:
:cactus: 🔬 I don't _love_ `me` as a var name. Can we use something more descriptive like `player`?